src/ray/gcs/gcs_server/gcs_monitor_server.cc

```diff
@@ -185,7 +187,16 @@ void GcsMonitorServer::HandleGetSchedulingStatus(
   PopulateResourceDemands(reply);
   PopulatePlacementGroupDemands(reply);
 
-  send_reply_callback(Status::OK(), nullptr, nullptr);
+  internal_kv_.Get("",
```

Comment:
Is it possible to assign a namespace for this key? This really looks bad.

Reply:
done